x/mongo/driver/operation.go

```diff
@@ -792,7 +792,18 @@ func (op Operation) Execute(ctx context.Context) error {
 			if moreToCome {
 				roundTrip = op.moreToComeRoundTrip
 			}
-			res, err = roundTrip(ctx, conn, *wm)
+
+			readOpts := []mnet.ReadOption{}
+			if maxTimeMS != 0 {
```

Comment:
What about the case where a user provides maxTimeMs in the command function, such as tailable awaitData cursors? In that case we omit maxTimeMS, but we should still rely on the pending read behavior.